### PR TITLE
#28 Design Registration Page and send request to backend

### PR DESCRIPTION
**Pull request by DanielWill-Baggs**
Built registration component in react and sent data to api endpoints using a fetch request while handling any response from the backend.

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,38 +1,65 @@
...
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+form.input>input {
```
Comment (OliverA100) [inline frontend/src/App.css:28]: Not certain if the >input is a typo or a CSS functionality, please let me know.

**Review by OliverA100 — COMMENTED** [review]
Everything looks good except for some doubts about a specific CSS statement. I have added a comment about it. Please come back to me with an answer. Thanks.

File: frontend/src/App.css (hunk 1 above)
Comment (DanielWill-Baggs) [reply frontend/src/App.css:28]: This is a CSS functionality and is correct!

Comment (OliverA100) [reply frontend/src/App.css:28]: Ok perfect.

**Review by OliverA100 — APPROVED** [review]
Everything clarified, approved for merging!

**Pull request merged**